packages/cli-kit/src/public/node/output.ts

```diff
@@ -245,20 +245,32 @@ export function collectLog(key: string, content: OutputMessage): void {
 }
 
 export const clearCollectedLogs = (): void => {
-  // console.log('clearCollectLogs')
   collectedLogs = {}
-  // console.log(collectedLogs)
 }
 
 /**
- * Ouputs information to the user.
+ * Outputs command result information to the user.
+ * Result messages don't get additional formatting.
+ * Note: By default, result messages are logged at info level to stdout.
+ *
+ * @param content - The content to be output to the user.
+ * @param logger - The logging function to use to output to the user.
+ */
+export function outputResult(content: OutputMessage, logger: Logger = consoleLog): void {
+  const message = stringifyMessage(content)
+  if (isUnitTest()) collectLog('info', content)
+  outputWhereAppropriate('info', logger, message)
+}
```

Comment:
Could this just be:

```suggestion
export function outputResult(content: OutputMessage, logger: Logger = consoleLog): void {
  output(content, 'info', logger)
}
```

(importing `output` of course)

Reply:
Yeah, thanks!